templates/vue/src/components/node-modal/ConditionsForm.vue

```diff
@@ -1,6 +1,9 @@
 <template>
   <div>
-    <b-form-group class="mb-3">
+    <b-form-group
+      class="mb-3"
+      description="Users who are not logged in won't be able to unlock this node."
```

Comment:
Can you change this to say "Please note: Currently, locked nodes cannot be unlocked by users who are not logged in."

Also, please make it so it appears more like a warning and not a description. I suggest not showing it before the checkbox is checked, again so it doesn't appear like the description for that checkbox.